s3/src/test/scala/akka/stream/alpakka/s3/scaladsl/S3IntegrationSpec.scala

```diff
@@ -471,7 +473,9 @@ trait S3IntegrationSpec
       result.lift(currentChunk) match {
         case Some(currentString) =>
           val newString = ByteString(s"\n${newAcc.toString()}")
-          if (currentString.length <= S3.MinChunkSize) {
+          //TODO: Performance here can be improved by using the same technique as in
+          //createStringCollectionContextWithMinChunkSizeRec to keep track of the current string size in bytes
+          if (currentString.toArray.length < S3.MinChunkSize) {
```

Comment:
There was a fencepost error which I realized when writing `createStringCollectionContextWithMinChunkSizeRec`. For this specific test it didn't make a difference since we were only checking the entire file and didn't care about chunks however for the `perform a chunked multi-part upload with the correct context` test this is relevant